rhel7-insights-client/insights-client.yml

```diff
@@ -56,8 +56,6 @@
       git:
         repo: "https://github.com/RedHatInsights/{{ item }}.git"
         dest: "/home/vagrant/{{ item }}"
-      args:
-        creates: "/home/vagrant/{{ item }}"
```

Comment:
This is needed so the playbook skips this step if you've mounted the repos or if they are already cloned into the VM. Perhaps I missed a git option, like `update: false`.